src/hooks/useGetBotUser.ts

```diff
@@ -2,21 +2,23 @@ import SendbirdChat, { ApplicationUserListQuery, User } from '@sendbird/chat';
 import useSendbirdStateContext from '@sendbird/uikit-react/useSendbirdStateContext';
 import { useEffect, useState } from 'react';
 
+import { useHashedKey } from '../context/HashedKeyContext';
+
 export function useGetBotUser(
   currentUser: User | null,
   sendbirdBotId: string
 ): User | null {
   const [botUser, setBotUser] = useState<User | null>(null);
   const store = useSendbirdStateContext();
+  const { hashedKey } = useHashedKey();
   const sb: SendbirdChat = store.stores.sdkStore.sdk;
 
   useEffect(() => {
-    // console.log('## useGetBotUser: ', { hashedKey: sendbirdBotId, id: currentUser?.userId });
     if (currentUser && sendbirdBotId) {
-      // console.log('## useGetBotUser hashedKey: ', hashedKey);
       const query: ApplicationUserListQuery = sb.createApplicationUserListQuery(
         {
-          userIdsFilter: [sendbirdBotId],
+          userIdsFilter:
+            hashedKey != null ? [sendbirdBotId, hashedKey] : [sendbirdBotId],
```

Comment:
It seems this is the only place the `hashedKey` is being used in https://github.com/sendbird/ai-bot-url-webdemo